pipeline_ukbiobank/cli/manual_correction.py

```diff
@@ -40,9 +41,10 @@ def get_parser():
         required=True,
         help=
         "R|Config yaml file listing images that require manual corrections for segmentation and vertebral "
-        "labeling. 'FILES_SEG' lists images associated with spinal cord segmentation"
+        "labeling. 'FILES_SEG' lists images associated with spinal cord segmentation "
         "and 'FILES_LABEL' lists images associated with vertebral labeling. "
-        "You can validate your yaml file at this website: http://www.yamllint.com/. Below is an example yaml file:\n"
+        "You can validate your yaml file at this website: http://www.yamllint.com/."
```

Comment:
just for consistency
```suggestion
        "You can validate your .yml file at this website: http://www.yamllint.com/."
```